BugsAndExploits/TntDuping.md

```diff
@@ -6,15 +7,21 @@ description: Explaination of tnt duplication
 # tnt duplication
 
 ## History
-TNT Duplication, or TNT Duping, was first discovered in 1.7 but was not used as it was slightly impractical before the addition of slime blocks. It gained popularity for the first time when upcoming technical Minecraft player ilmango released a video on YouTube entitled "The BEST Way To Make Perimeters in Minecraft," showcasing a slimestone flying machine that used flying TNT Dupers and liquid clearing machines to create a large hole fully automatically.
+TNT Duplication, or TNT Duping, was first discovered in 1.7 but was not used as it was slightly impractical before the addition of slime blocks.
```

Comment:
tnt duping this article is refering to was discoverd by myren in 1.9, it was later discoverd that you could do something similar in 1.7 and below but not 1.8

tnt duping is also extremly usefull even if you cant move the tnts with slime